internal/randutil/rand/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
The current pattern for license/copyright information is to copy the license into `THIRD-PARTY-NOTICES` with a reference to the code or package it references. Is that sufficient in this case or do we need to retain this `LICENSE` file in the package?